regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -46,6 +47,16 @@ var PreambleView = ChildView.extend({
     );
   },
 
+  openCitation: function(e) {
+    var $target = $(e.currentTarget);
+    var hash = $target.attr('href');
```

Comment:
Sorry, I'm not sure what's required here. Some things which may help:
* AJAX in footer navigation (which I believe also covers the navigation in the preamble): https://github.com/eregs/regulations-site/blob/ee5e24893916a9bb2eb361cc39df4323aa6b220e/regulations/static/regulations/js/source/views/main/section-footer-view.js#L22
* AJAX internal citations for non-notice-and-comment: https://github.com/eregs/regulations-site/blob/74ee00c3001454478cc95884b637464361424b75/regulations/static/regulations/js/source/views/main/reg-view.js#L67

If it looks like a rabbit hole, I think a quick win would be to take what you have but throw away all of the text before the hash in the href.